src/main/java/ch/uzh/ifi/seal/soprafs20/wordcheck/Stemmer.java

```diff
@@ -0,0 +1,387 @@
+package ch.uzh.ifi.seal.soprafs20.wordcheck;
+
+
+/*
+
```

Comment:
I was getting TLS handshake issue. So proceeded with algo only instead of external API